templates/helm/values.yaml.tpl

```diff
@@ -28,6 +28,10 @@ deployment:
   # Which priorityClassName to set?
   # See: https://kubernetes.io/docs/concepts/scheduling-eviction/pod-priority-preemption/#pod-priority
   priorityClassName: ""
+  hostNetwork: false
+  # Which dnsPolicy to set?
+  # See: https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pod-s-dns-policy
+  dnsPolicy:
```

Comment:
@kahirokunn  Looks like tests are failing. I suspect that this fields needs a default policy - what's the default DNS policy for pods in kubernetes?